peer.js

```diff
@@ -1,21 +1,22 @@
 const Handshake = require('./packets/clientbound/handshake.js');
 const localizePacket = require('./localize.js');
 const Packet = require('./packet.js');
+const log = require('loglevel')
 const protocolVersion = 404
 
 function handlePacket(packet, playerList, x, z, eidTable) {
-  var localizedPacket = localizePacket(packet, x, z, eidTable, 0)
-  playerList.notify(localizedPacket.loadIntoBuffer())
+  var localizedPacket = localizePacket.clientbound(packet, x, z, eidTable)
+  if(localizedPacket) { playerList.notify(localizedPacket.loadIntoBuffer()) }
```

Comment:
Currently this will always evaluate to true since the peer is only sending packets we need to see. However the localizePacket methods allow for the option of conditionally ignoring a packet, so we should be ready for that scenario